src/test/fuzz/addrman.cpp

```diff
@@ -27,7 +27,7 @@ class CAddrManDeterministic : public CAddrMan
 public:
     void MakeDeterministic(const uint256& random_seed)
     {
-        insecure_rand = FastRandomContext{random_seed};
+        WITH_LOCK(cs, insecure_rand = FastRandomContext{random_seed});
```

Comment:
21940: missing https://github.com/bitcoin/bitcoin/pull/21940/files#diff-593914ce009ce7e5a7aedc18eeefa2be2f0da0eb2c4ae19cee43abf4a9fd5717R106-R110, should be `partial`